tests/behat/context/BrowserContext.php

```diff
@@ -350,6 +350,38 @@ public function assertNoneOfTheElementsContain(string $selector, string $value):
     Assert::assertFalse($contains, "A element '".$selector."' contains '".$value."'");
   }
 
+  /**
+   * @Then /^one of the "(?P<selector>[^"]*)" elements should have value "(?P<value>(?:[^"]|\\")*)"$/
+   */
+  public function assertOneOfTheElementsHasValue(string $selector, string $value): void
```

Comment:
Why do we need this? I only find one element matching the #edit-description selector

Reply:
I thought it might be more reuseable if you could search multiple selectors. I can make it just search for one though

Reply:
Removed after applying suggested changes below